sor/src/test/java/com/bazaarvoice/emodb/sor/core/CompactorTest.java

```diff
@@ -75,12 +76,12 @@ public void testLegacyCompaction() {
         Delta delta3 = Deltas.mapBuilder().put("key2", "change").build();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "change" 11 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)